bitnami/node/templates/_helpers.tpl

```diff
@@ -86,7 +86,11 @@ Also, we can't use a single if because lazy evaluation is not an option
         {{- printf "%s/%s:%s" $registryName $repositoryName $tag -}}
     {{- end -}}
 {{- else -}}
-    {{- printf "%s/%s:%s" $registryName $repositoryName $tag -}}
+    {{- if .Values.image.registry -}}
```

Comment:
```suggestion
    {{- if $registryName -}}
```
we already have this variable using the proper value, use it just for consistency.